src/style/index.css

```diff
@@ -1306,4 +1306,39 @@ input[type="number"] {
     font-weight: 700;
     color: #2856ff;
     font-size: 1rem;
-}
\ No newline at end of file
+}
+
+
+
+@media print {
```

Comment:
Would it be possible to use tailwind's `print:*` modifier to apply these styles coming under `@media print` without using CSS?

Reply:
Hi @rithviknishad ,

I’ve updated the PR with Tailwind’s print: modifiers to handle print-specific styles instead of using traditional CSS. The changes include:

Applying padding and margin for print media.
Styling the header and footer for print.
Could you please review the changes and let me know if everything looks good or if there are any further adjustments needed?

Thanks!